views.py

```diff
@@ -246,9 +177,9 @@ def view_profile():
 
 
 @app.route('/trainers')
-def all_pokemon():
-    """A list of all Pokemon in Pokemon Go."""
-
+def all_trainers():
```

Comment:
I think you jumbled up some naming of view functions. See the other examples I highlighted. Minor error, but check it out.